client/app/components/EditVisualizationButton/QueryControlDropdown.jsx

```diff
@@ -67,7 +67,7 @@ export default function QueryControlDropdown(props) {
   );
 
   return (
-    <Dropdown trigger={["click"]} overlay={menu} overlayClassName="query-control-dropdown-overlay">
+    <Dropdown trigger={["click"]} overlay={menu} role="menu" overlayClassName="query-control-dropdown-overlay">
```

Comment:
Can you confirm this is necessary? The DOM reads like the `menu` attribute is going to be provided to the overlay without this.

Reply:
It is indeed!